sv_live_map_core/automation_window.py

```diff
@@ -93,6 +93,14 @@ def automation_work(self):
         self.target_found = True
         sys.exit()
 
+    def advance_date(self):
+        """Full process of dateskipping w/o checks for target_found"""
+        self.leave_to_home()
+        self.open_settings()
+        self.open_datetime()
+        self.skip_date()
+        self.reopen_game()
+
```

Comment:
might want to merge this method with ``full_dateskip`` via an optional argument for whether or not to verify ``self.target_found``